data/virt_autotest/guest_params_xml_files/sles_15_sp5_64_kvm_hvm_sev_es_x86_64.xml

```diff
@@ -84,7 +84,7 @@
     <guest_autoconsole>text</guest_autoconsole>
     <guest_noreboot></guest_noreboot>
     <guest_default_target>graphical</guest_default_target>
-    <guest_do_registration>true</guest_do_registration>
+    <guest_do_registration>false</guest_do_registration>
```

Comment:
Maybe it's better to have it true? It happens in the past that at RC or GMC phase, some bug fixes are posted in update channel already.

Reply:
There is authentication problem at the moment. Turning it on later might be better.

Reply:
Okay, then let's merge it as it is. BTW, for such cases, in future, it will be helpful to add a comment for the reason and link it  to either a TODO or a progress ticket so that it won't be forgotten.